### DISCUSSION TITLE
Add examples to how we can change the keybindings for the pager

### DISCUSSION THREAD

**Discussion by Dominiquini** [sachaos/viddy#148] [General]
In the README, there are examples of how we could change the keybindings for the time machine (right panel). But I could´t find how change the keybinds for the default pager!

Thanks.

**Comment by sachaos** [comment]
@Dominiquini Thank you.
Yes, that should be described in the README.
I'll convert this to issue.

issue: https://github.com/sachaos/viddy/issues/149
